pages/notes.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: page
+title:  "Notes"
```

Comment:
Should this just be part of the v1 spec page?

Reply:
I think this will apply to other versions of the spec, too. But if you have strong opinions...

Reply:
That's true, having a generic notes page makes sense.